packages/daisyui/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@rjsf/daisyui",
+  "version": "6.0.0-alpha.1",
+  "description": "Daisy UI components for react-jsonschema-form",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "scripts": {
+    "build": "tsc -b",
```

To improve ESM support you will need to change this to:
```suggestion
    "build:ts": "tsc -b tsconfig.build.json && tsc-alias -p tsconfig.build.json",
```

And add a `tsconfig.build.json` that looks like:
```
{
  "extends": "../../tsconfig.build.json",
  "compilerOptions": {
    "outDir": "./lib"
  },
  "files": [],
  "references": [
    {
      "path": "./src"
    }
  ],
  "tsc-alias": {
    "resolveFullPaths": true,
    "verbose": true,
  }
}
```